chapter_appendix_math/statistics_vn.md

```diff
@@ -119,7 +124,8 @@ There can be many ways to compute an estimator of a parameter $\hat{\theta}_n$.
 In this section, we introduce three common methods to evaluate and compare estimators: the mean squared error, the standard deviation, and statistical bias.
 -->
 
-*dịch đoạn phía trên*
+Có thể có nhiều cách để tính toán một bộ ước lượng cho một tham số $\hat{\theta}_n$.
+Trong phần này, ta sẽ điểm qua ba phương thức phổ biến để đánh giá và so sánh các bộ ước lượng: sai số toàn phương trung bình, độ lệch chuẩn và độ chệch thống kê.
```

Comment:
```suggestion
Trong phần này, ta sẽ điểm qua ba phương thức phổ biến để đánh giá và so sánh các bộ ước lượng: sai số bình phương trung bình, độ lệch chuẩn và độ chệch thống kê.
```

Mình cần thống nhất một cách dịch. Anh nhớ từ này đã được dịch là "bình phương", trong cuốn MLY cũng dịch như vậy.

Reply:
@tiepvupsu Cuốn MYL do dịch lâu quá nên em lên wiki tìm nhanh thì được cách dịch này

https://vi.m.wikipedia.org/wiki/Sai_s%E1%BB%91_to%C3%A0n_ph%C6%B0%C6%A1ng_trung_b%C3%ACnh

Reply:
Anh thấy rõ ràng "sai số toàn phương trung bình" vừa sai thứ tự các từ, vừa hơi xa vời với ý nghĩa của MSE. Từ "squared" ở đây chỉ đơn giản là "bình phương" thôi, từ "toàn phương" là dịch của từ "quadratic".

Reply:
Nếu em đọc kỹ thì sẽ hiểu "dạng toàn phương" (quadratic form) trong toán học nó có ý nghĩa khác. Hoặc như đây là ví dụ về quy hoạch toàn phương:
https://vi.wikipedia.org/wiki/Quy_ho%E1%BA%A1ch_to%C3%A0n_ph%C6%B0%C6%A1ng

Anh nghĩ mình chỉ dùng wiki để tham khảo, không nên coi đó là một chuẩn mực.

Cụm này dịch đúng phải là "trung bình bình phương sai số", tức là mình lấy trung bình các bình phương của sai số. Còn "sai số toàn phương trung bình" là ngược hoàn toàn rồi.